go/teams/transactions.go

```diff
@@ -592,7 +601,9 @@ func (tx *AddMemberTx) ReAddMemberToImplicitTeam(ctx context.Context, uv keybase
 	}
 
 	if hasPUK {
-		tx.addMember(uv, role)
+		if err := tx.addMember(uv, role, botSettings); err != nil {
+			return err
+		}
```

Comment:
is the existing comment below about "only posting one link" and "tried to create more than one link" ok with botSettings?

Reply:
ah good catch. added some test coverage here and updated the comment -- the number of links is not important but implicit teams cannot have a removal without an addition which is the real reason the crypto members cannot be swept